.github/workflows/java-ut.yml

```diff
@@ -0,0 +1,30 @@
+name: java
+
+on: pull_request
+
+jobs:
+  unit_tests:
+    name: Java Unit Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('java/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Test
+        working-directory: ./java
+        run: mvn --batch-mode -Dmaven.test.failure.ignore=true test
```

Comment:
Is this flag intended? `-Dmaven.test.failure.ignore=true`
It will cause the action to succeed eventhough tests might be failing.

Reply:
Removed the flag